src/tests.c

```diff
@@ -4,6 +4,8 @@
  * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
  ***********************************************************************/
 
+#define _POSIX_C_SOURCE 199309L /* for clock_gettime() */
+
```

Comment:
That's annoying that test sources have to be modified. See https://github.com/bitcoin-core/secp256k1/pull/1734#issuecomment-3447767148.